src/stash/src/postgres.rs

```diff
@@ -65,6 +66,21 @@ CREATE TABLE uppers (
 );
 ";
 
+struct PreparedStatements {
+    select_epoch: Statement,
+    iter_key: Statement,
+    since: Statement,
+    upper: Statement,
+    collection: Statement,
+    iter: Statement,
+    seal: Statement,
+    compact: Statement,
+    consolidate_consolidation: Statement,
+    consolidate_insert: Statement,
+    consolidate_delete: Statement,
+    update_many: Statement,
+}
+
 /// A Stash whose data is stored in a Postgres database. The format of the
 /// tables are not specified and should not be relied upon. The only promise is
 /// stability. Any changes to the table schemas will be accompanied by a clear
```

Comment:
I guess this was already there, but it looks like this comment was cut off?

Reply:
Will fix in another PR.